README.md

```diff
@@ -113,12 +113,14 @@ Out of the box, release-it has sane defaults, and [plenty of options](./config/r
 (only) the options to override in a configuration file. This is where release-it looks for configuration:
 
 - `.release-it.json`
-- `.release-it.js` (export the configuration object: `module.exports = {}`)
+- `.release-it.js` (or `.cjs`; export the configuration object: `module.exports = {}`)
 - `.release-it.yaml` (or `.yml`)
 - `.release-it.toml`
 - `package.json` (in the `release-it` property)
 
-Use `--config` to use another path for the configuration file. An example `.release-it.json`:
+Use `--config` to use another path for the configuration file.
+
+An example `.release-it.json`:
```

Comment:
If you could please undo this change from the PR I'm happy to merge it!

Reply:
Sorry - I actually thought (for whatever reason) that was a typo and there was supposed to be a line break there so I changed it while I was in the area. I've reverted it back to the way it was.